app/components-react/widgets/GameWidget.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { useState } from 'react';
+import { IWidgetState, useWidget, WidgetModule } from './common/useWidget';
+import { WidgetLayout } from './common/WidgetLayout';
+import { $t } from 'services/i18n';
+import { metadata } from 'components-react/shared/inputs/metadata';
+import { TextInput, ColorInput, createBinding, SliderInput } from 'components-react/shared/inputs';
+import Form from 'components-react/shared/inputs/Form';
+import { Menu } from 'antd';
+
+type TGameType = 'tic-tac-toe';
+
+interface ITicTacToeOptions {
+  background_color: string;
+  border_color: string;
+  chat_marker_color: string;
+  ai_marker_color: string;
+  chat_win_marker_color: string;
+  ai_win_marker_color: string;
+  chat_won_game_message: string;
+  chat_lost_game_message: string;
+  draw_game_message: string;
+  chat_marker: string;
+  ai_marker: string;
+  game_ended_message_duration: number;
+  chat_turn_message: string;
+  ai_turn_message: string;
+  cannot_play_here: string;
+}
+
+interface IGameWidgetState extends IWidgetState {
+  data: {
+    settings: {
+      decision_poll_timer: number;
+      trigger_command: string;
+      current_game: TGameType;
+      no_input_received_message: string;
+      restarting_game_message: string;
+      available_games: TGameType[];
+      game_options: {
+        'tic-tac-toe': ITicTacToeOptions;
+      };
+    };
+  };
+}
+
+export function GameWidget() {
+  const { isLoading, bind } = useGameWidget();
+
+  const [activeTab, setActiveTab] = useState('general');
+
+  return (
+    <WidgetLayout>
+      <Menu onClick={e => setActiveTab(e.key)} selectedKeys={[activeTab]}>
+        <Menu.Item key="general">{$t('General Settings')}</Menu.Item>
+        <Menu.Item key="game">{$t('Game Settings')}</Menu.Item>
+      </Menu>
+      <Form>
+        {!isLoading && activeTab === 'general' && (
+          <>
+            <SliderInput
+              label={$t('Chat Decision Time')}
+              tooltip={{
+                title: $t(
+                  "The duration in seconds to collect chat's responses before processing passing them to the game.",
```

Comment:
Seems like copy here isn't quite right.  Maybe `processing` was meant to be removed?

Reply:
yea this was just copy-pasted from the site, can change